lib/iex/lib/iex/pry.ex

```diff
@@ -111,7 +111,7 @@ defmodule IEx.Pry do
   @doc """
   Annotate quoted expression with line-by-line `IEx.Pry` debugging steps.
 
-  It expected the `quoted` expression to annotate, a `condition` that controls
+  It expects the `quoted` expression to annotate, a boolean `condition` that controls
```

Comment:
`condition` is a quoted expression that evaluates to a boolean.

Reply:
Yep good catch, fixed!